.github/workflows/pytests.yml

```diff
@@ -42,9 +42,14 @@ jobs:
 
       - name: Install latest ASE from gitlab
         run: |
-          python3 -m pip install git+https://gitlab.com/ase/ase.git
+          python3 -m pip install ase 
```

Comment:
Should we specify a version, now that it might start mattering? `ase==3.23.0`, I guess?

Reply:
I initially thought that it might be good to keep up with the latest pip release, but it might be nicer to choose when to update manually. Added the version (`v3.23.0`)

Reply:
Actually, it might be interesting to have to CI jobs, one that needs to pass, which tests on a version of ASE we are committing to support, and another on the latest gitlab HEAD, which is just informational for us.  But not part of this PR.